src/plugins/intel_cpu/src/emitters/x64/jit_load_store_emitters.cpp

```diff
@@ -81,6 +81,7 @@ jit_load_emitter::jit_load_emitter(dnnl::impl::cpu::x64::jit_generator *host, dn
     prepare_table();
     load_size_ = load_num * src_prc.size();
     v_len_elt_ = get_vec_length() / exec_prc.size();
+    m_snippets_segfault_detector = false;
```

Comment:
I understand why we have to always set `m_snippets_segfault_detector` for the emitters which are used in snippets emitters. But I still believe that we need to avoid changes in non snippets related emitters.

I think we can fulfill these conditions in the following way:
1. Set `m_snippets_segfault_detector` to false in jit_emitters class
2. Introduce a setter for this field
3. Enable segfault detector (if the corresponding env variable is set) for all snippets emitters in one place: in [LinearIR::init_emitters](https://github.com/openvinotoolkit/openvino/blob/master/src/common/snippets/src/lowered/linear_ir.cpp#L192).

What do you think about this solution?

Reply:
great idea to enable from snippets, done.